web/cobrands/shropshire/assets.js

```diff
@@ -18,10 +18,8 @@ var defaults = {
         }
     },
     asset_type: 'spot',
-    asset_id_field: 'central_as',
```

Comment:
I think you wanted to keep this line, renamed as the attribute was? This is used for knowing if two assets being compared by the code are the same one, for reselecting after layer change or what have you.